javascript/src/webhook.test.ts

```diff
@@ -36,64 +36,67 @@ class TestPayload {
   }
 }
 
-test('missing id raises error', () => {
```

Comment:
Could you maybe move the prettier run to its own PR? Same with the Python changes.
Not the end of the world if too annoying this time, but at least for next time then...